data/campaigns/Son_Of_The_Black_Eye/scenarios/16_The_Coward.cfg

```diff
@@ -174,15 +174,13 @@
             [else]
                 [role]
                     side=1
-                    x,y=recall,recall
+                    search_recall_list=only
                     type=Orcish Warlord,Orcish Slurbow,Saurian Flanker,Troll Warrior,Orcish Warrior,Orcish Crossbowman,Saurian Oracle,Saurian Soothsayer,Saurian Ambusher,Troll,Troll Rocklobber,Orcish Slayer
                     role=initial_backup
+                    [auto_recall]
+                        x,y=44,9
+                    [/auto_recall]
                 [/role]
-
-                [recall]
-                    role=initial_backup
-                    x,y=44,9
-                [/recall]
```

Comment:
The role is never used. One can use the [recall] tag as well with multiple types, and only the first one matching will be recalled.